### PR TITLE
Initial content published

### PR DESCRIPTION
**Pull request by hjscherer**
Moving the content from our engineering repo into this public repo

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Owners of repository
+/.github/CODEOWNERS @calcof
+/README.md @calcof
+
+# Owners for introduction docs
+/docs/introduction/ @calcof @heoelri
```
Comment (sebader) [inline .github/CODEOWNERS]: not that I want to get rid of Heyko, nor do I want to see my name there. But...why him on the intro? :D

File: CONTRIBUTE.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+To add or edit content within the AlwaysOn repositories, please take a fork of a repository to iterate on changes before subsequently opening a Pull Request (PR) to get your forked branch merged into the main branch for that AlwaysOn repository. Your PR will be reviewed by core engineers working on the AlwaysOn project, and once approved, your content accessible to everybody.
+
+> **Important!** Please make sure that your PR is focused on a specific area of AlwaysOn to facilitate a targeted review, as this will speed up the process to get your changes merged into our repository.
+
+## Content Structure
+
+The AlwaysOn project is separated into **3** different repositories:
+
+* [AlwaysOn](./README.md): contains the AlwaysOn design methodology, covering the design pattern ad approach to guide readers to defining a target AlwaysOn architecture.
```
Comment (sebader) [inline CONTRIBUTE.md]: lets switch from * to - to stay consistent

File: CONTRIBUTE.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  * Overarching topics are documented as separate markdown documents within the `/docs/` directory.
+
+* [AlwaysOn-Foundational-Online](http://github.com/azure/alwayson-foundational-online): contains the AlwaysOn foundational reference implementation intended for online scenarios that are public-facing and do not require private network connectivity to a surrounding organizational technical estate.
```
Comment (sebader) [inline CONTRIBUTE.md]: I guess merging this also means that we should change the names of the other repos now?!

File: CONTRIBUTE.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  * `/docs/` contains the majority of documentation, covering the design approach and detailed documentation to accompany the reference implementation.
```
Comment (sebader) [inline CONTRIBUTE.md]: should we make these deep links?

File: CONTRIBUTE.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  * `/.ado/pipelines` contains the Azure DevOps pipelines to build and deploy the reference implementation.
+
+* [AlwaysOn-Foundational-Connected](http://github.com/azure/alwayson-foundational-connected): contains the AlwaysOn foundational reference implementation intended for private scenarios that require integration with an organizational technical estate for either public-facing or internal-facing workloads.
+  * `/docs/` contains the majority of documentation, covering the design approach and detailed documentation to accompany the reference implementation.
+  * `/src/` contains all source code and technical artifacts for the reference implementation along with low level implementation documentation.
+  * `/.ado/pipelines` contains the Azure DevOps pipelines to build and deploy the reference implementation.
+
+## Documentation Conventions
+
+* Each source code component within the reference implementation repositories has it's own `README.md` file which explains how that particular component works, how it is supposed to be used, and how it may interact with other aspects of the AlwaysOn solution.
```
Comment (sebader) [inline CONTRIBUTE.md]: does this need to be a bullet point?

File: README.md [hunk 6]
```diff
@@ -1,33 +1,46 @@
...
+
+* [Foundational Reference Implementation | Online](https://github.com/azure/alwayson-foundational-online) - Everything required to understand and build a copy of the foundational reference implementation intended for online scenarios that are public-facing and do not require private network connectivity to a surrounding organizational technical estate.
+
+* [Foundational Reference Implementation | Connected](https://github.com/azure/alwayson-foundational-connected) - Everything required to understand and build a copy of the foundational reference implementation intended for private scenarios that require integration with an organizational technical estate for either public-facing or internal-facing workloads.
+
+## Helpful Information
+
+* [Getting Started](./docs/Getting-Started.md) outlines the process and required steps to deploy AlwaysOn in your environment, including preparing the Azure DevOps pipelines. It should be read in tandem with the [Reference Implementation Guide](./docs/reference-implementation/README.md).
```
Comment (sebader) [inline README.md]: this repo does not have a getting started guide (anymore). but maybe it should get one?! Which basically should say (again): "If you want to deploy online, go here, if you want to deploy connected, go there, etc...)"

File: docs/README.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+  * [Application Design](./design-guidelines/App-Design.md)
+  * [Application Platform](./design-guidelines/App-Platform.md)
+  * [Data Platform](./design-guidelines/Data-Platform.md)
+  * [Health Modeling and Observability](./design-guidelines/Health-Modeling.md)
+  * [Deployment and Testing](./design-guidelines/Deployment-Testing.md)
+  * [Networking and Connectivity](./design-guidelines/Networking.md)
+  * [Security](./design-guidelines/Security.md)
+  * [Operational Procedures](./design-guidelines/Operational-Procedures.md)
+
+## AlwaysOn Reference Implementation Guide
```
Comment (sebader) [inline docs/README.md]: this repo does not have the RI folder

File: docs/design-methodology/Operational-Procedures.md [hunk 8]
```diff
@@ -0,0 +1,175 @@
...
+
+- Manual operational changes to update components should be avoided and only considered by emergency exception.
+  - Ensure a process exists to reconciliate any manual changes back into the source repository, avoiding drift and issue recurrence.
+  - Establish an automated housekeeping procedure to [remove old image versions from Azure Container Registry](https://docs.microsoft.com/azure/container-registry/container-registry-auto-purge).
+
+---
+
+|Previous Page|Next Page|
+|:--|:--|
+|[Security](./Security.md) |[AlwaysOn Reference Implementation](https://github.com/Azure/AlwaysOn/blob/main/docs/reference-implementation/README.md)
```
Comment (sebader) [inline docs/design-methodology/Operational-Procedures.md]: dead link

File: docs/README.md [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+    * [Networking Design Decisions](./reference-implementation/Networking-Design-Decisions.md)
+    * [Custom Domains](./reference-implementation/Networking-Custom-Domains.md)
+  * Security
+    * [Azure Active Directory B2C Provisioning](/src/config/identity/B2C-Provisioning.md)
+  * Operational Procedures
+    * [Operational Procedures](./reference-implementation/OpProcedures-Operational-Procedures.md)
+    * [Key and Secret Rotation](./reference-implementation/OpProcedures-KeyRotation.md)
+* [ESLZ Alignment](./reference-implementation/ESLZ-Alignment.md)
+
+## Source Code README Files
```
Comment (sebader) [inline docs/README.md]: also no src folder

**Review by sebader — CHANGES_REQUESTED** [review]

File: CONTRIBUTE.md (hunk 2 above)
Comment (hjscherer) [reply CONTRIBUTE.md]: done

File: CONTRIBUTE.md (hunk 3 above)
Comment (hjscherer) [reply CONTRIBUTE.md]: yes, will do

File: docs/design-methodology/Operational-Procedures.md (hunk 8 above)
Comment (hjscherer) [reply docs/design-methodology/Operational-Procedures.md]: done

File: docs/README.md [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+
+- [Landing Page](/README.md)
+
+## Introduction to AlwaysOn
+
+- [Introduction](./introduction/README.md)
+
+## AlwaysOn Design Methodology
+
+- [How to use the AlwaysOn Design Methodology](./design-guidelines/README.md)
```
Comment (sebader) [inline docs/README.md]: dead links. folder name needs updating

File: docs/design-methodology/Principles.md [hunk 11]
```diff
@@ -0,0 +1,103 @@
...
+
+The AlwaysOn architectural framework presented within this repository is underpinned by 5 key design principles which serve as a compass for subsequent design decisions across technical domains and the critical design areas. Readers are strongly advised to familiarize themselves with these principles to better understand their impact and the trade-offs associated with non-adherence.
+
+1. **Maximum Reliability** - Fundamental pursuit of the most reliable solution, ensuring trade-offs are properly understood.
+1. **Sustainable Performance and Scalability** - Design for scalability across the end-to-end solution without performance bottlenecks.
+1. **Operations by Design** - Engineered to last with robust and assertive operational management.
+1. **Cloud-Native Design** - Focus on using native platforms services to minimize operational burdens, while mitigating known gaps.
+1. **Always Secure** - Design for end-to-end security to maintain application stability and ensure availability.
+
+[![AlwaysOn Design Principles](/docs/media/alwayson-design-principles.png "AlwaysOn Design Principles")](./Principles.md)
```
Comment (sebader) [inline docs/design-methodology/Principles.md:11]: dead picture

Comment (sebader) [reply docs/design-methodology/Principles.md:11]: image

File: docs/design-methodology/Design-Areas.md [hunk 12]
```diff
@@ -0,0 +1,102 @@
...
+- [Data Platform](./Data-Platform.md)
+- [Health Modeling](./Health-Modeling.md)
+- [Deployment and Testing](./Deployment-Testing.md)
+- [Networking and Connectivity](./Networking.md)
+- [Security](./Security.md)
+- [Operational Procedures](./Operational-Procedures.md)
+
+These 8 critical design areas will be explored at length within ensuing pages, for which critical review considerations and design recommendations are provided along with their broader design impact across other areas. Ultimately, the design areas are interrelated and decisions made within one area can impact or influence decisions across the entire design, so readers are encouraged to use the provided design guidance to navigate the key design decisions.
+
+[![AlwaysOn Design Areas](/docs/media/alwayson-design-areas.png "AlwaysOn Design Areas")](./Design-Areas.md)
```
Comment (sebader) [inline docs/design-methodology/Design-Areas.md:16]: dead picture

File: docs/design-methodology/Design-Areas.md [hunk 13]
```diff
@@ -0,0 +1,102 @@
...
+- Unblock with preview services or by taking dependencies on the Azure platform roadmap
+- Only take a dependency on committed services and features; validate roadmap dependencies with Microsoft engineering product groups
+
+### Azure Landing Zone Integration
+
+[Azure Landing Zones](https://github.com/azure/cloud-adoption-framework/ready/landing-zone/) provides prescriptive architectural guidance to define a reliable and scalable shared-service platform for enterprise Azure deployments with requisite centralised governance. 
+
+AlwaysOn can integrate seamlessly within an Azure Landing Zone, and is deployable within both the *Online* or *Corp. Connected* Landing Zone formats as demonstrated within the image below.
+
+[![AlwaysOn and Landing Zone Integration](./images/alwayson-landing-zones.gif "AlwaysOn Landing Zone Integration")](./Design-Areas.md)
```
Comment (sebader) [inline docs/design-methodology/Design-Areas.md]: dead pic

File: docs/design-methodology/App-Design.md [hunk 14]
```diff
@@ -0,0 +1,319 @@
+# Application Design
```
Comment (sebader) [inline docs/design-methodology/App-Design.md:1]: a couple of dead pics in this doc

**Review by sebader — CHANGES_REQUESTED** [review]

File: docs/design-methodology/Design-Areas.md (hunk 12 above)
Comment (hjscherer) [reply docs/design-methodology/Design-Areas.md:16]: case sensitive :-)

File: docs/design-methodology/Design-Areas.md (hunk 13 above)
Comment (hjscherer) [reply docs/design-methodology/Design-Areas.md]: done

File: docs/design-methodology/App-Design.md (hunk 14 above)
Comment (hjscherer) [reply docs/design-methodology/App-Design.md:1]: done

File: README.md [hunk 15]
```diff
@@ -1,33 +1,48 @@
...
+
+- [Foundational Reference Implementation | Connected](https://github.com/azure/alwayson-foundational-connected) - Everything required to understand and build a copy of the foundational reference implementation intended for private scenarios that require integration with an organizational technical estate for either public-facing or internal-facing workloads.
+
+## Helpful Information
+
+- Each reference implementation provides a getting *Getting Started guide* in the repository:
+  - Foundational Reference Implementation Online | [Getting started guide](https://github.com/Azure/AlwaysOn-foundational-online/blob/main/docs/reference-implementation/Getting-Started.md)
+  - Foundational Reference Implementation Connected | [Getting started guide](https://github.com/Azure/AlwaysOn-foundational-connected/blob/main/docs/reference-implementation/Getting-Started.md)
+
+- [Frequently Asked Questions](./docs/FAQ.md): captures responses to common issues and challenges associated with using the AlwaysOn project.
```
Comment (heoelri) [inline README.md]: the `FAQ.md` does not exist in this branch/repo. I still like the idea but I think it wasn't updated for a while now. Do we want to keep it or drop it here?

File: docs/README.md [hunk 16]
```diff
@@ -0,0 +1,34 @@
...
+## AlwaysOn Design Methodology
+
+- [How to use the AlwaysOn Design Methodology](./design-methodology/README.md)
+- [Design Principles](./design-methodology/Principles.md)
+- [Design Areas](./design-methodology/Design-Areas.md)
+  - [Reference Architecture](./design-methodology/Design-Areas.md#reference-architecture)
+  - [Cross Cutting Concerns](./design-methodology/Design-Areas.md#cross-cutting-concerns)
+  - [Application Design](./design-methodology/App-Design.md)
+  - [Application Platform](./design-methodology/App-Platform.md)
+  - [Data Platform](./design-design-methodology/Data-Platform.md)
```
Comment (heoelri) [inline docs/README.md]:
```suggestion
  - [Data Platform](./design-methodology/Data-Platform.md)
```

File: docs/README.md [hunk 17]
```diff
@@ -0,0 +1,34 @@
...
+  - [Health Modeling and Observability](./design-design-methodology/Health-Modeling.md)
+  - [Deployment and Testing](./design-design-methodology/Deployment-Testing.md)
+  - [Networking and Connectivity](./design-design-methodology/Networking.md)
+  - [Security](./design-design-methodology/Security.md)
```
Comment (heoelri) [inline docs/README.md]:
```suggestion
  - [Health Modeling and Observability](./design-methodology/Health-Modeling.md)
  - [Deployment and Testing](./design-methodology/Deployment-Testing.md)
  - [Networking and Connectivity](./design-methodology/Networking.md)
  - [Security](./design-methodology/Security.md)
```

File: README.md (hunk 15 above)
Comment (hjscherer) [reply README.md]: Create a FAQ.md file with some initial questions. We can always add more, but I like the idea too

**Review by heoelri — APPROVED** [review]
LGTM

**Review by sebader — APPROVED** [review]
nice work, thank you!

**Review by nielsams — APPROVED** [review]

**Review by CalCof — APPROVED** [review]

File: README.md [hunk 18]
```diff
@@ -22,13 +22,13 @@ AlwaysOn is an open source project that provides a **prescriptive architectural
 
 ## Helpful Information
 
-- Each reference implementation provides a getting *Getting Started guide* in the repository:
-  - Foundational Reference Implementation Online | [Getting started guide](https://github.com/Azure/AlwaysOn-foundational-online/blob/main/docs/reference-implementation/Getting-Started.md)
-  - Foundational Reference Implementation Connected | [Getting started guide](https://github.com/Azure/AlwaysOn-foundational-connected/blob/main/docs/reference-implementation/Getting-Started.md)
+The foundational reference implementations are separated within a dedicated repository containing all relevant documentation and technical artefacts, along with a *getting started guide*:
```
Comment (sebader) [inline README.md]:
```suggestion
The foundational reference implementations are separated within dedicated repositories containing all relevant documentation and technical artifacts, along with a *getting started guide*:
```

**Review by sebader — CHANGES_REQUESTED** [review]

**Review by sebader — APPROVED** [review]

**Pull request merged**